datastore/google/cloud/datastore/__init__.py

```diff
@@ -16,14 +16,26 @@
 
 You'll typically use these to get started with the API:
 
-.. code-block:: python
+.. testsetup:: constructors
 
-  from google.cloud import datastore
+  import os
+  os.environ['GOOGLE_CLOUD_PROJECT'] = u'my-project'
```

Comment:
1. This isn't a global `testsetup`, just for a single `doctest` block
1. I was thinking about this yesterday and just using `...` where the project goes is probably better